modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/model/SupportedLanguage.java

```diff
@@ -23,33 +23,33 @@ public class SupportedLanguage {
 
 	@JsonProperty("language")
 	public String getLanguage() {
-		return _language;
+		return language;
 	}
 
 	@JsonProperty("name")
 	public String getName() {
-		return _name;
+		return name;
 	}
 
 	@JsonProperty("supports_formality")
 	public Boolean getSupportsFormality() {
-		return _supportsFormality;
+		return supportsFormality;
 	}
 
 	public void setLanguage(String language) {
-		_language = language;
+		this.language = language;
 	}
 
 	public void setName(String name) {
-		_name = name;
+		this.name = name;
 	}
 
 	public void setSupportsFormality(Boolean supportsFormality) {
-		_supportsFormality = supportsFormality;
+		this.supportsFormality = supportsFormality;
 	}
 
-	private String _language;
-	private String _name;
-	private Boolean _supportsFormality;
+	protected String language;
+	protected String name;
+	protected Boolean supportsFormality;
```

Comment:
Add a constructor and make the fields private. If the setters are no longer necessary, remove them.